flaps/flaps_machines.go

```diff
@@ -22,6 +23,38 @@ func (f *Client) sendRequestMachines(ctx context.Context, method, endpoint strin
 	return f._sendRequest(ctx, method, endpoint, in, out, headers)
 }
 
+const (
+	LaunchCapacityErrStr = "no capacity available in"
+)
+
+type LaunchCapacityErr struct {
+	region string
+}
+
+var launchCapacityErrRe *regexp.Regexp = regexp.MustCompile(fmt.Sprintf(`%s (?P<region>[a-zA-Z0-9]*)`, LaunchCapacityErrStr))
```

Comment:
Curious if we can update flyd/flaps to return errors as JSON with a CODE field and some extra text or data and just pass that straight through. Thoughts?

Reply:
done!